### PR TITLE
added contacts to contacts.json

### PR DESCRIPTION
**Pull request by daviddassau**
- already had navbar 
- added google auth, with uid set in $rootScope
- created JSON file with contacts
- Deployed to Firebase: https://angular-contacts-db03c.firebaseapp.com/#!/login

### CODE REVIEW

File: db/contacts.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+    "contacts": [
+        {
```
Comment (morecallan) [inline db/contacts.json]: This should be an object of objects with keys for each of the contacts (just like it will be from Firebase).

**Review by morecallan — COMMENTED** [review]
@daviddassau - Looks like you have an update for your JSON file. Additionally, phone number should be a number and not a string. Once you update that, I'll merge it.

**Comment by morecallan** [comment]
@daviddassau - also, go ahead and add uid to the contacts.json 🍔

**Comment by daviddassau** [comment]
@morecallan 
- Made phone number a number, not a string
- Added uid to each contact
- Edited JSON file to have contact0, contact1, etc

**Pull request merged**